tsconfig.json

```diff
@@ -14,7 +14,9 @@
     "noUnusedParameters": true,
     "outDir": "dist/esm",
     "sourceMap": true,
-    "target": "es2017"
+    "target": "es2017",
+    "strictNullChecks": false,
+    "skipLibCheck": true,
   },
```

Comment:
Added these because of https://github.com/maplibre/ngx-maplibre-gl/issues/89#issuecomment-1561674827